File: GTTS/INL.TrainingService.Models/GetTrainingEventRosterInGroups_Result.cs
using System.Collections.Generic;

namespace INL.TrainingService.Models
{
    public class GetTrainingEventRosterInGroups_Result : IGetTrainingEventRosterInGroups_Result
    {
        public long TrainingEventID { get; set; }
        public List<IGetTrainingEventRosterGroups_Item> RosterGroups { get; set; }
    }
}

File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.TrainingService.Models/get-training-event-roster-in-groups_result.ts


import { IGetTrainingEventRosterGroups_Item } from './iget-training-event-roster-groups_item';
import { IGetTrainingEventRosterInGroups_Result } from './iget-training-event-roster-in-groups_result';

export class GetTrainingEventRosterInGroups_Result implements IGetTrainingEventRosterInGroups_Result {
  
	public TrainingEventID: number = 0;
	public RosterGroups?: IGetTrainingEventRosterGroups_Item[];
  
}






